app/models/manageiq/providers/amazon/manager_mixin.rb

```diff
@@ -143,28 +108,5 @@ def discover_queue(access_key_id, secret_access_key)
     def discover_from_queue(access_key_id, secret_access_key)
       discover(access_key_id, MiqPassword.decrypt(secret_access_key))
     end
```

Comment:
@juliancheal - and what about this?

Reply:
So I thought `discover_queue` was queue related, but yeah realise now it just calls `discover` indirectly. Thanks.